desktop.css

```diff
@@ -0,0 +1,592 @@
+    width: 80%;
+    height: 80vh;
+    justify-content: center;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    position: absolute;
+    width: 90%;
+    height: 30%;
+    left: 20%;
+    top: calc(35% - 268px / 2);
+  }
+
+  .main-text1 {
+    width: 90%;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .main-text2 {
+    width: 90%;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .socials-icons {
+    position: absolute;
+    left: 20%;
+    top: calc(66% - 268px / 2);
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 25% 8%;
+    width: 100%;
+    position: relative;
+    height: 100%;
+    box-sizing: border-box;
+    left: 0;
+    z-index: 0;
+  }
+
+  .card-works-1 {
+    width: 100%;
+    height: 50%;
+    flex-direction: row;
+    gap: 22px;
+    padding: 24px;
+    margin-bottom: 5%;
+  }
+
+  .card-works-2 {
+    width: 100%;
+    height: 50%;
+    flex-direction: row-reverse;
+    gap: 22px;
+    padding: 24px;
+    margin-bottom: 5%;
+  }
+
+  .work-snapshot {
+    width: 50%;
+    height: auto;
+    border-radius: 8px;
+  }
+
+  .work-block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px;
+    gap: 24px;
+    width: 50%;
+    height: auto;
+    box-sizing: border-box;
+  }
+
+  .primary-text {
+    width: 100%;
+    height: auto;
+  }
+
+  .project-title {
+    width: 100%;
+    height: auto;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .client {
+    width: 100%;
+    height: auto;
+  }
+
+  .client p {
+    width: 100%;
+    height: auto;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .role {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 0;
+    width: 100%;
+    height: auto;
+  }
+
+  .role p {
+    width: 100%;
+    height: auto;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .year {
+    width: 100%;
+    height: auto;
+  }
+
+  .year p {
+    width: 100%;
+    height: auto;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .project-details {
+    width: 100%;
+    height: auto;
+    font-size: 16px;
+  }
+
+  .tags {
+    width: 100%;
+    height: auto;
+  }
+
+  .actions {
+    width: 100%;
+    height: auto;
+    align-items: flex-start;
+  }
+
+  .actions button {
+    gap: 10px;
+    left: 0;
+    top: 16px;
+  }
+
+  .about-myself {
+    flex-direction: row;
+    width: 100%;
+    height: 100%;
+    padding-left: 8%;
+    padding-right: 8%;
+    padding-bottom: 20%;
+    box-sizing: border-box;
+    border-top-right-radius: 150px;
+    margin-top: -12%;
+    z-index: 1;
+  }
```

Comment:
This section contains 2 blocks. The left one takes 40% and the right one take the remaining 60% of the page's width. However, I see that your design has partitioned them into two, each occupying 50%. Kindly check and correct this anomaly in order to match the actual design. 

### Your design 
![Screenshot from 2023-03-03 19-23-07](https://user-images.githubusercontent.com/65233860/222812088-42d08f2e-5b45-4dcc-b293-237098bda7dd.png)

### Actual design

![Screenshot from 2023-03-03 19-23-25](https://user-images.githubusercontent.com/65233860/222812103-8cc57435-b2d8-46c0-aa47-77338fa6452c.png)